File: src/domain/influencer.py
from typing import List
from pydantic import BaseModel
from .audience import Audience
from .tone_of_voice import ToneOfVoice
from .post import Post

class Influencer(BaseModel):
    id: str
    audience: Audience
    relevant_works_count: int
    relevant_events_count: int
    tone_of_voice: ToneOfVoice
    posts: List[Post]
    

File: node/src/models/Influencer.ts
import type { Audience } from "./Audience";
import type { Post } from "./Post";
import { ToneOfVoice } from "./ToneOfVoice";

export interface Influencer {
  id: string;
  name: string;
  audience: Audience;
  relevant_works_count: number;
  relevant_events_count: number;
  tone_of_voice: typeof ToneOfVoice[keyof typeof ToneOfVoice];
  posts: Post[];
}
